.github/workflows/ci.yaml

```diff
@@ -16,7 +16,7 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
-      - uses: ./
+      - uses: open-turo/action-renovate@${{ github.event.pull_request.head.sha }}
```

Comment:
CI fails with
> Unrecognized named-value: 'github'. Located at position 1 within expression: github.event.pull_request.head.sha

Reply:
Yeah, still testing the best way to have this working (I should have created the PR as draft)